### ISSUE TITLE
New bugfix release

### ISSUE THREAD

**Issue by DArtagan** [sibson/redbeat#265]
tl;dr: if possible, please cut a new bugfix pypi release.

Longer story; there are a few open Github issues which have been fixed in `main`:
* #251 & #255, when timezones are present.  Luckily, @ElijahL opened PR #256 with a fix back in August, which has been merged to `main`.
* #264 with versions of Celery newer than 5.2.7, when `FixedOffset` was removed.  Happily @jkseppan fixed this back in May, with merged PR #245.

While waiting for a release, in our project we've kept RedBeat pinned to `<2.1.1`, which has meant keeping celery pinned to `<5.3.0`.  Python 3.12 was released in October, which removed a deprecated function that Celery was using (`EntryPoints.get()` in the `importlib-metadata` package).  Recent versions of Celery have removed that call, but we remain pinned to the older version.  We would love to upgrade to the next major release, and if the code that's already in `main` could be released, then we could let the versions of `celery-redbeat` and `celery` float free!

If there's anything preventing the release of a new version to pypi, and additional Python-capable hands could help, please let me know where I can lend a hand.

**Comment by sibson** [comment]
https://pypi.org/project/celery-redbeat/2.2.0/

**Comment by DArtagan** [comment]
This brought me great joy this week, thank you.
